### DISCUSSION TITLE
First-party cookies blocked

### DISCUSSION THREAD

**Discussion by helenafranca** [GoogleChromeLabs/ps-analysis-tool#631] [Support Forum] [answered]
In the example below, several 1P cookies are blocked. In my understanding, they are blocked because www.voeazul.com.br is a different domain than voeazul.com.br. Nevertheless, if domains are different, why does PSAT consider these cookies as first-party cookies?
![Screenshot 2024-04-18 6 06 11 PM](https://github.com/GoogleChromeLabs/ps-analysis-tool/assets/85519105/a3600ea2-511e-4fb7-9461-ae1fecf22f08)

**Comment by amedina** [comment, marked as answer]
<img width="3120" alt="Screenshot 2024-04-18 at 2 42 31 PM" src="https://github.com/GoogleChromeLabs/ps-analysis-tool/assets/506089/8d238d0e-c957-460b-8b50-2e6ef16b5a45">
The scenario in these cases is not that the domain of the cookie is a 3P domain, but the domain of the cookie does not match exactly the domain of the top-level site (i.e. Domain mismatch). Several factors can play a role here. Look at the values of the `SameSite` attribute as well as the `Path` attribute. Some of the blocked 1P cookies have `SameSite=Strict`, others have different values in the `Path`, and others have `www` as the prefix of the top-level domain whereas the top-level domain does not have `www`.

**Comment by amedina** [comment]
Also notice that most of these cookies are blocked only on some of the requests going out, as indicated by the orange upwards arrow. This means that the cookies are not fully blocked.

**Reply by helenafranca** [reply]
I find it interesting that in many cases, due to the SameSite attribute, these cookies are already broken today.
